scripts/install.sh

```diff
@@ -7,8 +7,15 @@
 
 set -u
 
+BINARY_DOWNLOAD_PREFIX="https://github.com/apollographql/router/releases/download"
+
+# Router version defined in apollo-router's Cargo.toml
+# Note: Change this line manually during the release steps.
+PACKAGE_VERSION="v0.16.0"
```

Comment:
could this be passed to the script as an environment variable or an argument instead?